docs/transmission.md

```diff
@@ -0,0 +1,89 @@
+# Transmission model
+
+## Overview
+
+The transmission model controls person-to-person transmission between an
+infectious agent and other susceptible agents. Agents become infectious immediately
+upon successful infection attempt, though they may have a transmission probability of
+0 for some time (mimicking an exposed compartment). Once infectious, agents have a pre-assigned
+number of infection attempts. Infection attempt times are drawn from the generation interval.
+Once there are no more infection attempts remaining for the agent, we argue that they do not
+transition to a recovered compartment but rather back to a susceptible compartment but
+with high immunity.
+
+## Workflow
+
+The transmission model is centered around a key workflow:
+1. Watch for an agent becoming infectious (S --> I transition).
+2. For the infectious agent, draw a number of infection attempts. This can be from the
+$R_i$ distribution or the contact distribution.
+3. Draw an infection time for the agent. This consists of taking an ordered draw from the generation
+interval. The math for this is abstracted away under a generic `get_next_infection_time` method.
+4. Draw a susceptible contact from the agent's contact group. Contacts could be drawn uniformly
+from the contact group or with weight based on person characteristics. The actual
+drawing of a contact can be abstracted into a `get_contact` method. Evaluate whether
```

Comment:
Wondering what this should look like for our specific example, although we don't need to elaborate on this right now